UnitySDK/Assets/ML-Agents/Editor/Tests/EditModeTestInternalBrainTensorGenerator.cs

```diff
@@ -55,109 +54,109 @@ public void GenerateBatchSize()
         {
             var inputTensor = new TensorProxy();
             var alloc = new TensorCachingAllocator();
-            var batchSize = 4;
+            const int batchSize = 4;
             var generator = new BatchSizeGenerator(alloc);
             generator.Generate(inputTensor, batchSize, null);
-            Assert.IsNotNull(inputTensor.Data);
-            Assert.AreEqual(inputTensor.Data[0], batchSize);
+            Assert.IsNotNull(inputTensor.data);
+            Assert.AreEqual(inputTensor.data[0], batchSize);
             alloc.Dispose();
         }
-        
+
         [Test]
         public void GenerateSequenceLength()
         {
             var inputTensor = new TensorProxy();
             var alloc = new TensorCachingAllocator();
-            var batchSize = 4;
+            const int batchSize = 4;
             var generator = new SequenceLengthGenerator(alloc);
             generator.Generate(inputTensor, batchSize, null);
-            Assert.IsNotNull(inputTensor.Data);
-            Assert.AreEqual(inputTensor.Data[0], 1);
+            Assert.IsNotNull(inputTensor.data);
+            Assert.AreEqual(inputTensor.data[0], 1);
             alloc.Dispose();
         }
-        
+
         [Test]
         public void GenerateVectorObservation()
         {
             var inputTensor = new TensorProxy()
             {
-                Shape = new long[] {2, 3}
+                shape = new long[] {2, 3}
             };
-            var batchSize = 4;
+            const int batchSize = 4;
             var agentInfos = GetFakeAgentInfos();
             var alloc = new TensorCachingAllocator();
             var generator = new VectorObservationGenerator(alloc);
             generator.Generate(inputTensor, batchSize, agentInfos);
-            Assert.IsNotNull(inputTensor.Data);
-            Assert.AreEqual(inputTensor.Data[0, 0], 1);
-            Assert.AreEqual(inputTensor.Data[0, 2], 3);
-            Assert.AreEqual(inputTensor.Data[1, 0], 4);
-            Assert.AreEqual(inputTensor.Data[1, 2], 6);
+            Assert.IsNotNull(inputTensor.data);
+            Assert.AreEqual(inputTensor.data[0, 0], 1);
+            Assert.AreEqual(inputTensor.data[0, 2], 3);
+            Assert.AreEqual(inputTensor.data[1, 0], 4);
+            Assert.AreEqual(inputTensor.data[1, 2], 6);
             alloc.Dispose();
         }
-        
+
         [Test]
         public void GenerateRecurrentInput()
         {
             var inputTensor = new TensorProxy()
             {
-                Shape = new long[] {2, 5}
+                shape = new long[] {2, 5}
             };
-            var batchSize = 4;
+            const int batchSize = 4;
```

Comment:
What's the general guideline for when to use `var`? You move some things to it, and others (like here) away from it.

Reply:
because [`const`](https://stackoverflow.com/questions/6132412/why-cant-i-declare-a-constant-using-var-in-c)

Reply:
The general guideline is to use it when the variable left type matches the right type.